platforms/android/aar-creating/build_java_shared_aar.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+from os import path
+import os
+import shutil
+import subprocess
+
+from build_static_aar import fill_template
+
+
+COPY_FROM_SDK_TO_ANDROID_PROJECT = [
+    ["sdk/native/jni/include", "OpenCV/src/main/cpp/include"],
+    ["sdk/java/src/org", "OpenCV/src/main/java/org"],
+    ["sdk/java/res", "OpenCV/src/main/res"]
+]
+
+ABIS = ["arm64-v8a", "armeabi-v7a", "x86", "x86_64"] # if you want to change it, you also need to change Android project template
+
+COPY_FROM_SDK_TO_APK = [
+    ["sdk/native/libs/<ABI>/libopencv_java4.so", "jni/<ABI>/libopencv_java4.so"],
+    ["sdk/native/libs/<ABI>/libopencv_java4.so", "prefab/modules/opencv_java4/libs/android.<ABI>/libopencv_java4.so"],
+]
+
+ANDROID_PROJECT_TEMPLATE_DIR = "OpenCVAndroidProject"
```

Comment:
The script might be executed outside of OpenCV. Please use full path.

Reply:
Maybe use __file__ constant?